conf/machine/include/stm32mp1.inc

```diff
@@ -1,5 +1,7 @@
 DEFAULTTUNE = "cortexa7thf-neon-vfpv4"
 
+SOC_FAMILY = "stm32mp1"
+
```

Comment:
Nice, can you rebase on top-of warrior ?

Reply:
it's done